include/rdma/fabric.h

```diff
@@ -340,6 +340,7 @@ struct fi_fabric_attr {
 	char			*prov_name;
 	uint32_t		prov_version;
 	uint32_t		api_version;
+	char			*comp_list;
```

Comment:
field behaves differently from other getinfo fields -- it is used as a filter, not a request mechanism.   E.g. can we specify which components to select, not select?  Random idea was to use something like "+" or "-" to request or disable a feature...